Comparing source compatibility of /devhome/git/hygieia-subversion-scm-collector/target/subversion-collector-3.0.2-SNAPSHOT.jar against /Users/kpf557/.m2/repository/com/capitalone/dashboard/subversion-collector/3.0.1/subversion-collector-3.0.1.jar
===  UNCHANGED CLASS: PUBLIC com.capitalone.dashboard.Application  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC Application()
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
	===  UNCHANGED ANNOTATION: org.springframework.boot.autoconfigure.SpringBootApplication
===  UNCHANGED CLASS: PUBLIC com.capitalone.dashboard.collector.DefaultSubversionClient  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: com.capitalone.dashboard.collector.SubversionClient
	===  UNCHANGED CONSTRUCTOR: PUBLIC DefaultSubversionClient(com.capitalone.dashboard.collector.SubversionSettings)
		===  UNCHANGED ANNOTATION: org.springframework.beans.factory.annotation.Autowired
	===  UNCHANGED METHOD: PUBLIC java.util.List getCommits(com.capitalone.dashboard.model.SubversionRepo, long)
	===  UNCHANGED METHOD: PUBLIC long getRevisionClosestTo(java.lang.String, java.util.Date)
	===  UNCHANGED ANNOTATION: org.springframework.stereotype.Component
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT com.capitalone.dashboard.collector.SubversionClient  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List getCommits(com.capitalone.dashboard.model.SubversionRepo, long)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT long getRevisionClosestTo(java.lang.String, java.util.Date)
===  UNCHANGED CLASS: PUBLIC com.capitalone.dashboard.collector.SubversionCollectorTask  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Runnable
	===  UNCHANGED CONSTRUCTOR: PUBLIC SubversionCollectorTask(org.springframework.scheduling.TaskScheduler, com.capitalone.dashboard.repository.BaseCollectorRepository, com.capitalone.dashboard.repository.SubversionRepoRepository, com.capitalone.dashboard.repository.CommitRepository, com.capitalone.dashboard.repository.ComponentRepository, com.capitalone.dashboard.collector.SubversionClient, com.capitalone.dashboard.collector.SubversionSettings)
		===  UNCHANGED ANNOTATION: org.springframework.beans.factory.annotation.Autowired
	===  UNCHANGED METHOD: PUBLIC void collect(com.capitalone.dashboard.model.Collector)
	===  UNCHANGED METHOD: PUBLIC com.capitalone.dashboard.model.Collector getCollector()
	===  UNCHANGED METHOD: PUBLIC com.capitalone.dashboard.repository.BaseCollectorRepository getCollectorRepository()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getCron()
	===  UNCHANGED ANNOTATION: org.springframework.stereotype.Component
===  UNCHANGED CLASS: PUBLIC com.capitalone.dashboard.collector.SubversionSettings  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC SubversionSettings()
	===  UNCHANGED METHOD: PUBLIC int getCommitThresholdDays()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getCron()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getPassword()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getUsername()
	===  UNCHANGED METHOD: PUBLIC void setCommitThresholdDays(int)
	===  UNCHANGED METHOD: PUBLIC void setCron(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void setPassword(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void setUsername(java.lang.String)
	===  UNCHANGED ANNOTATION: org.springframework.stereotype.Component
	===  UNCHANGED ANNOTATION: org.springframework.boot.context.properties.ConfigurationProperties
		===  UNCHANGED ELEMENT: prefix=subversion
===  UNCHANGED CLASS: PUBLIC com.capitalone.dashboard.model.SubversionRepo  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String LATEST_REV
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String URL
	===  UNCHANGED CONSTRUCTOR: PUBLIC SubversionRepo()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC long getLatestRevision()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getRepoUrl()
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC void setLatestRev(long)
	===  UNCHANGED METHOD: PUBLIC void setRepoUrl(java.lang.String)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT com.capitalone.dashboard.repository.SubversionRepoRepository  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: org.springframework.data.repository.PagingAndSortingRepository
	===  UNCHANGED INTERFACE: org.springframework.data.repository.Repository
	===  UNCHANGED INTERFACE: com.capitalone.dashboard.repository.BaseCollectorItemRepository
	===  UNCHANGED INTERFACE: org.springframework.data.repository.CrudRepository
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List findEnabledSubversionRepos(org.bson.types.ObjectId)
		===  UNCHANGED ANNOTATION: org.springframework.data.mongodb.repository.Query
			===  UNCHANGED ELEMENT: value={ &apos;collectorId&apos; : ?0, enabled: true}
	===  UNCHANGED METHOD: PUBLIC ABSTRACT com.capitalone.dashboard.model.SubversionRepo findSubversionRepo(org.bson.types.ObjectId, java.lang.String)
		===  UNCHANGED ANNOTATION: org.springframework.data.mongodb.repository.Query
			===  UNCHANGED ELEMENT: value={ &apos;collectorId&apos; : ?0, options.url : ?1}
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.springframework.boot.loader.archive.Archive  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC Archive()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection getEntries()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.springframework.boot.loader.archive.Archive getFilteredArchive(org.springframework.boot.loader.archive.Archive$EntryRenameFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getMainClass()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.jar.Manifest getManifest()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List getNestedArchives(org.springframework.boot.loader.archive.Archive$EntryFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT STATIC org.springframework.boot.loader.archive.Archive$Entry  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.springframework.boot.loader.util.AsciiBytes getName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isDirectory()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT STATIC org.springframework.boot.loader.archive.Archive$EntryFilter  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean matches(org.springframework.boot.loader.archive.Archive$Entry)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT STATIC org.springframework.boot.loader.archive.Archive$EntryRenameFilter  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.springframework.boot.loader.util.AsciiBytes apply(org.springframework.boot.loader.util.AsciiBytes, org.springframework.boot.loader.archive.Archive$Entry)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.archive.ExplodedArchive  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExplodedArchive(java.io.File, boolean)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExplodedArchive(java.io.File)
	===  UNCHANGED METHOD: PUBLIC java.util.Collection getEntries()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.archive.Archive getFilteredArchive(org.springframework.boot.loader.archive.Archive$EntryRenameFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.jar.Manifest getManifest()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PROTECTED org.springframework.boot.loader.archive.Archive getNestedArchive(org.springframework.boot.loader.archive.Archive$Entry)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.List getNestedArchives(org.springframework.boot.loader.archive.Archive$EntryFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.archive.FilteredArchive  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC FilteredArchive(org.springframework.boot.loader.archive.Archive, org.springframework.boot.loader.archive.Archive$EntryFilter)
	===  UNCHANGED METHOD: PUBLIC java.util.Collection getEntries()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.archive.Archive getFilteredArchive(org.springframework.boot.loader.archive.Archive$EntryRenameFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getMainClass()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC java.util.jar.Manifest getManifest()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.List getNestedArchives(org.springframework.boot.loader.archive.Archive$EntryFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.archive.JarFileArchive  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarFileArchive(org.springframework.boot.loader.jar.JarFile)
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarFileArchive(java.io.File, java.net.URL)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarFileArchive(java.io.File)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.Collection getEntries()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.archive.Archive getFilteredArchive(org.springframework.boot.loader.archive.Archive$EntryRenameFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.jar.Manifest getManifest()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PROTECTED org.springframework.boot.loader.archive.Archive getNestedArchive(org.springframework.boot.loader.archive.Archive$Entry)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.List getNestedArchives(org.springframework.boot.loader.archive.Archive$EntryFilter)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.data.ByteArrayRandomAccessData  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: org.springframework.boot.loader.data.RandomAccessData
	===  UNCHANGED CONSTRUCTOR: PUBLIC ByteArrayRandomAccessData(byte[])
	===  UNCHANGED CONSTRUCTOR: PUBLIC ByteArrayRandomAccessData(byte[], long, long)
	===  UNCHANGED METHOD: PUBLIC java.io.InputStream getInputStream(org.springframework.boot.loader.data.RandomAccessData$ResourceAccess)
	===  UNCHANGED METHOD: PUBLIC long getSize()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.data.RandomAccessData getSubsection(long, long)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.springframework.boot.loader.data.RandomAccessData  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.io.InputStream getInputStream(org.springframework.boot.loader.data.RandomAccessData$ResourceAccess)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT long getSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.springframework.boot.loader.data.RandomAccessData getSubsection(long, long)
===  UNCHANGED ENUM: PUBLIC STATIC FINAL org.springframework.boot.loader.data.RandomAccessData$ResourceAccess  (compatible)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.springframework.boot.loader.data.RandomAccessData$ResourceAccess PER_READ
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.springframework.boot.loader.data.RandomAccessData$ResourceAccess ONCE
	===  UNCHANGED METHOD: PUBLIC STATIC org.springframework.boot.loader.data.RandomAccessData$ResourceAccess valueOf(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.springframework.boot.loader.data.RandomAccessData$ResourceAccess[] values()
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.data.RandomAccessDataFile  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: org.springframework.boot.loader.data.RandomAccessData
	===  UNCHANGED CONSTRUCTOR: PUBLIC RandomAccessDataFile(java.io.File)
	===  UNCHANGED CONSTRUCTOR: PUBLIC RandomAccessDataFile(java.io.File, int)
	===  UNCHANGED METHOD: PUBLIC void close()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.io.File getFile()
	===  UNCHANGED METHOD: PUBLIC java.io.InputStream getInputStream(org.springframework.boot.loader.data.RandomAccessData$ResourceAccess)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC long getSize()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.data.RandomAccessData getSubsection(long, long)
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.springframework.boot.loader.ExecutableArchiveLauncher  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExecutableArchiveLauncher(org.springframework.boot.loader.JavaAgentDetector)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExecutableArchiveLauncher()
	===  UNCHANGED CONSTRUCTOR: PROTECTED ExecutableArchiveLauncher(org.springframework.boot.loader.archive.Archive)
	===  UNCHANGED METHOD: PROTECTED java.lang.ClassLoader createClassLoader(java.net.URL[])
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED FINAL org.springframework.boot.loader.archive.Archive getArchive()
	===  UNCHANGED METHOD: PROTECTED java.util.List getClassPathArchives()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.String getMainClass()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED ABSTRACT boolean isNestedArchive(org.springframework.boot.loader.archive.Archive$Entry)
	===  UNCHANGED METHOD: PROTECTED void postProcessClassPathArchives(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.InputArgumentsJavaAgentDetector  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: org.springframework.boot.loader.JavaAgentDetector
	===  UNCHANGED CONSTRUCTOR: PUBLIC InputArgumentsJavaAgentDetector()
	===  UNCHANGED METHOD: PUBLIC boolean isJavaAgentJar(java.net.URL)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.jar.Handler  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC Handler()
	===  UNCHANGED CONSTRUCTOR: PUBLIC Handler(org.springframework.boot.loader.jar.JarFile)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarFile getRootJarFileFromUrl(java.net.URL)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PROTECTED java.net.URLConnection openConnection(java.net.URL)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC void setUseFastConnectionExceptions(boolean)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.jar.JarEntry  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: java.lang.Cloneable
	===  UNCHANGED INTERFACE: java.util.zip.ZipConstants
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarEntry(org.springframework.boot.loader.jar.JarEntryData)
	===  UNCHANGED METHOD: PUBLIC java.util.jar.Attributes getAttributes()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.security.cert.Certificate[] getCertificates()
	===  UNCHANGED METHOD: PUBLIC java.security.CodeSigner[] getCodeSigners()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarEntryData getSource()
	===  UNCHANGED METHOD: PUBLIC java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
===  UNCHANGED CLASS: PUBLIC FINAL org.springframework.boot.loader.jar.JarEntryData  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarEntryData(org.springframework.boot.loader.jar.JarFile, byte[], java.io.InputStream)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes getComment()
	===  UNCHANGED METHOD: PUBLIC int getCompressedSize()
	===  UNCHANGED METHOD: PUBLIC long getCrc()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.data.RandomAccessData getData()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC byte[] getExtra()
	===  UNCHANGED METHOD: PUBLIC int getMethod()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes getName()
	===  UNCHANGED METHOD: PUBLIC int getSize()
	===  UNCHANGED METHOD: PUBLIC long getTime()
	===  UNCHANGED METHOD: PUBLIC boolean isDirectory()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.springframework.boot.loader.jar.JarEntryFilter  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.springframework.boot.loader.util.AsciiBytes apply(org.springframework.boot.loader.util.AsciiBytes, org.springframework.boot.loader.jar.JarEntryData)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.jar.JarFile  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: java.io.Closeable
	===  UNCHANGED INTERFACE: java.lang.AutoCloseable
	===  UNCHANGED INTERFACE: java.lang.Iterable
	===  UNCHANGED INTERFACE: java.util.zip.ZipConstants
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarFile(java.io.File)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC void close()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.Enumeration entries()
	===  UNCHANGED METHOD: PUBLIC java.util.zip.ZipEntry getEntry(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarFile getFilteredJarFile(org.springframework.boot.loader.jar.JarEntryFilter[])
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.io.InputStream getInputStream(java.util.zip.ZipEntry)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarEntry getJarEntry(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarEntryData getJarEntryData(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarEntryData getJarEntryData(org.springframework.boot.loader.util.AsciiBytes)
	===  UNCHANGED METHOD: PUBLIC java.util.jar.Manifest getManifest()
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getName()
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarFile getNestedJarFile(java.util.zip.ZipEntry)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.jar.JarFile getNestedJarFile(org.springframework.boot.loader.jar.JarEntryData)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PROTECTED FINAL org.springframework.boot.loader.data.RandomAccessDataFile getRootJarFile()
	===  UNCHANGED METHOD: PUBLIC java.net.URL getUrl()
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
	===  UNCHANGED METHOD: PUBLIC java.util.Iterator iterator()
	===  UNCHANGED METHOD: PUBLIC STATIC void registerUrlProtocolHandler()
	===  UNCHANGED METHOD: PUBLIC int size()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.JarLauncher  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PROTECTED JarLauncher(org.springframework.boot.loader.archive.Archive)
	===  UNCHANGED CONSTRUCTOR: PUBLIC JarLauncher()
	===  UNCHANGED METHOD: PROTECTED boolean isNestedArchive(org.springframework.boot.loader.archive.Archive$Entry)
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
	===  UNCHANGED METHOD: PROTECTED void postProcessClassPathArchives(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.springframework.boot.loader.JavaAgentDetector  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isJavaAgentJar(java.net.URL)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.LaunchedURLClassLoader  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: java.io.Closeable
	===  UNCHANGED INTERFACE: java.lang.AutoCloseable
	===  UNCHANGED CONSTRUCTOR: PUBLIC LaunchedURLClassLoader(java.net.URL[], java.lang.ClassLoader)
	===  UNCHANGED METHOD: PUBLIC java.net.URL findResource(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.util.Enumeration findResources(java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.net.URL getResource(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.util.Enumeration getResources(java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PROTECTED java.lang.Class loadClass(java.lang.String, boolean)
		===  UNCHANGED EXCEPTION: java.lang.ClassNotFoundException
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.springframework.boot.loader.Launcher  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED FIELD: PROTECTED java.util.logging.Logger logger
	===  UNCHANGED CONSTRUCTOR: PUBLIC Launcher()
	===  UNCHANGED METHOD: PROTECTED FINAL org.springframework.boot.loader.archive.Archive createArchive()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.ClassLoader createClassLoader(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.ClassLoader createClassLoader(java.net.URL[])
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.Runnable createMainMethodRunner(java.lang.String, java.lang.String[], java.lang.ClassLoader)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED ABSTRACT java.util.List getClassPathArchives()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED ABSTRACT java.lang.String getMainClass()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void launch(java.lang.String[])
	===  UNCHANGED METHOD: PROTECTED void launch(java.lang.String[], java.lang.String, java.lang.ClassLoader)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.MainMethodRunner  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED INTERFACE: java.lang.Runnable
	===  UNCHANGED CONSTRUCTOR: PUBLIC MainMethodRunner(java.lang.String, java.lang.String[])
	===  UNCHANGED METHOD: PUBLIC void run()
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.PropertiesLauncher  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PATH
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ARGS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String SET_SYSTEM_PROPERTIES
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CONFIG_NAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String MAIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String HOME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CONFIG_LOCATION
	===  UNCHANGED CONSTRUCTOR: PUBLIC PropertiesLauncher()
	===  UNCHANGED METHOD: PROTECTED java.lang.ClassLoader createClassLoader(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.String[] getArgs(java.lang.String[])
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.util.List getClassPathArchives()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.io.File getHomeDirectory()
	===  UNCHANGED METHOD: PROTECTED java.lang.String getMainClass()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String toCamelCase(java.lang.CharSequence)
===  UNCHANGED CLASS: PUBLIC FINAL org.springframework.boot.loader.util.AsciiBytes  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC AsciiBytes(java.lang.String)
	===  UNCHANGED CONSTRUCTOR: PUBLIC AsciiBytes(byte[])
	===  UNCHANGED CONSTRUCTOR: PUBLIC AsciiBytes(byte[], int, int)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes append(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes append(org.springframework.boot.loader.util.AsciiBytes)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes append(byte[])
	===  UNCHANGED METHOD: PUBLIC boolean endsWith(org.springframework.boot.loader.util.AsciiBytes)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC int length()
	===  UNCHANGED METHOD: PUBLIC boolean startsWith(org.springframework.boot.loader.util.AsciiBytes)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes substring(int)
	===  UNCHANGED METHOD: PUBLIC org.springframework.boot.loader.util.AsciiBytes substring(int, int)
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.springframework.boot.loader.util.SystemPropertyUtils  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PLACEHOLDER_PREFIX
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PLACEHOLDER_SUFFIX
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String VALUE_SEPARATOR
	===  UNCHANGED CONSTRUCTOR: PUBLIC SystemPropertyUtils()
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getProperty(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getProperty(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getProperty(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String resolvePlaceholders(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String resolvePlaceholders(java.util.Properties, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.springframework.boot.loader.WarLauncher  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	===  UNCHANGED CONSTRUCTOR: PROTECTED WarLauncher(org.springframework.boot.loader.archive.Archive)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WarLauncher()
	===  UNCHANGED METHOD: PUBLIC boolean isNestedArchive(org.springframework.boot.loader.archive.Archive$Entry)
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
